### PR TITLE
Shell checker: Show error and read more button

### PR DESCRIPTION
**Pull request by worksofliam**
### Changes

Handles #1337.

* Adds error messages (`stderr`) to shell error modal
* Show 'Read more' button to open docs page with more info regarding this error

### Checklist

* [x] have tested my change
* [x] updated relevant documentation
* [ ] Remove any/all `console.log`s I added
* [ ] eslint is not complaining
* [ ] have added myself to the contributors' list in [CONTRIBUTING.md](https://github.com/halcyon-tech/vscode-ibmi/blob/master/CONTRIBUTING.md)
* [ ] **for feature PRs**: PR only includes one feature enhancement.

### CODE REVIEW

File: src/api/IBMi.ts [hunk 1]
```diff
@@ -180,16 +176,33 @@ export default class IBMi {
         const checkShellText = `This should be the only text!`;
         const checkShellResult = await this.sendCommand({
           command: `echo "${checkShellText}"`,
-          directory: `.` // Will use whatever the user profile default is
+          directory: `/` // Will use root as the working directory for this command
```
Comment (chrjorgensen) [inline src/api/IBMi.ts:179]: The `directory` has no effect and should be removed - the SSH server is changing directory to the non-existing user home directory before running the command and will return `Could not chdir to home directory /home/xxxxxxx: No such file or directory`

```suggestion
          command: `echo "${checkShellText}"`
        });
        if (checkShellResult.code === 0 && checkShellResult.stdout.split(`\n`)[0] !== checkShellText) {
```

**Review by chrjorgensen — CHANGES_REQUESTED** [review]
@worksofliam Your change was not enough - the SSH server is the one trying to change to the users home directory at login and setting an error message in `stderr` - and error code 0...!!

I have made a suggestion for a fix, that will work. It basically says to only check the shell output in `stdout` if no errors were found.

**Comment by worksofliam** [comment]
@chrjorgensen 

Your suggestion is more fitting and I've made a change so the shell check only looks at the the `stdout` (as it should). We already have logic to warn the user if their home directory is bad! 

image

**Comment by angelorpa** [comment]
another possibility could be used an SQL query to get user profile info, something like this:

`
SELECT USER_NAME, HOME_DIRECTORY, PASE_SHELL_PATH 
FROM QSYS2.USER_INFO
WHERE USER_NAME = 'USER_NAME'
`

if the HOME_DIRECTORY field is not empty so check the directory path value exist

**Comment by worksofliam** [comment]
@angelorpa That's a little bit out of context for this PR, but thanks for the idea nonetheless!

**Review by chrjorgensen — APPROVED** [review]
@worksofliam Looking good!

**Pull request merged**